### PR TITLE
target_gen and tests

### PR DESCRIPTION
**Pull request by bbarzgaran**
Updated target generation and its tests:
- Target gen has a zero vector check flag
- Updated target gen tests to reflect changes in other libraries and flag changes
- Added a flag test
Small updates to sun vector and aero drag tests (changing simulink files to 2016b)

### CODE REVIEW

**Comment by tpreynolds** [comment]
- There are no flags in the library -- maybe you uploaded the wrong version.
- For test 3, you're inputing a zero vector, but then there is no check on the flag output (that doesn't seem to be part of the library). 
    -- There should be an if-else condition in the matlab test file that checks the flag is set to the right thing, and indicates whether the test is passed or failed.
- There are unconnected signal lines in both test libraries -- fix this.
- There is some warning about algebraic loops when I run test 2. Needs to be checked.
- Did you verify that the DCM computed from the commented function matches up with the library when given the same inputs?

File: adcs/sw/components/adcs_sim/matlab/sim_init.m [hunk 1]
```diff
@@ -26,7 +26,7 @@
 fsw_params.bdot     = init_bdot_controller(fsw_params);
 
 % Load sim and set params
-run_time    = 360;
```
Comment (tpreynolds) [inline adcs/sw/components/adcs_sim/matlab/sim_init.m:29]: no need to commit this file if the only thing you've changed is the run time.

**Comment by bbarzgaran** [comment]
Updated target gen lib to reflect tolerance and colinear vectors check in TRIAD
Updated test files

**Comment by tpreynolds** [comment]
- Why are we checking the angle between vel_unit and sc2gs_unit?  This never gets used
- It seems inefficient to be computing the unit vectors, checking the angle, and then re-computing the unit vectors later. The checks should just be integrated inside the main if-statement, not a separate thing.

I think its okay for now since the library functions properly. Worth opening an issue to make the code more efficient at a later date.

**Pull request merged**